Cargo.toml

```diff
@@ -37,7 +37,7 @@ thiserror = "1.0"
 pasta-msm = { version = "0.1.4" }
 
 [dev-dependencies]
-criterion = "0.3.1"
+criterion = { version = "0.5.1", features = ["html_reports"] }
```

Comment:
Version 0.5.1 is not yet compatible pprof, which I introduce in #183. It would be great to only upgrade to 0.4 for now.

Reply:
I see, thank you.